userguide/tutorials/email-notification-plugin.adoc

```diff
@@ -61,12 +63,36 @@ The email-notification-plugin requires some additional database tables. To creat
 [[plugin_configuration]]
 == Plugin Configuration
 
-In order to use the email notification plugin, a tenant needs to be configured with the SMTP properties required for sending emails. Additionally, the tenant can also be configured with the events for which emails should be sent. In addition to the per-tenant configuration, the plugin also allows a more granular account-level configuration for the set of emails to be sent for the particular account. Thus, either the tenant or the account needs to be configured with the events for which the email needs to be sent.
+In order to use the email notification plugin, you need to configure either SMTP or AWS SES settings. Below are the steps to configure both:
+
+=== AWS SES Configuration
+To enable AWS SES for email notifications, add the following properties into killbill.properties. The default sender's email must be verified in AWS SES.
+
+[source,bash]
+org.killbill.email.notification.via.ses=true
+org.killbill.mail.from=<Default sender's email>
+org.killbill.aws.region=<AWS region, default region is us-east-1>
+
+AWS SES requires authentication for sending emails. System-wide environment variables can be configured for all users by appending them to the /etc/environment file. Open the `/etc/environment` file in a text editor with root privileges, set the following variables:
+
+[source,bash]
+AWS_ACCESS_KEY_ID=<AWS access key ID>
+AWS_SECRET_ACCESS_KEY=<AWS access secret>
+
+
+Reboot the system to apply the changes.
+
+
+[NOTE]
+*Note:* Ensure that the credentials used have the necessary permissions, specifically `ses:SendEmail`, on the resource arn:aws:ses:<region>:<account>:identity/<sender's email>.
+
+=== SMTP Configuration
+A tenant needs to be configured with the SMTP properties required for sending emails. Additionally, the tenant can also be configured with the events for which emails should be sent. In addition to the per-tenant configuration, the plugin also allows a more granular account-level configuration for the set of emails to be sent for the particular account. Thus, either the tenant or the account needs to be configured with the events for which the email needs to be sent.
```

Comment:
Move the text `In addition to the per-tenant configuration, the plugin also allows a more granular account-level configuration for the set of emails to be sent for the particular account. Thus, either the tenant or the account needs to be configured with the events for which the email needs to be sent.` to the `Configure Events` section below.

Reply:
I agree, but from what I read, the account configuration is now moved to the `Configure Events` section, hence I suggested moving the above text to the `Configure Events`. If you disagree, feel free to leave it as it is and merge the PR.